inference-engine/thirdparty/clDNN/tests/test_cases/fusings_gpu_test.cpp

```diff
@@ -7791,3 +7793,247 @@ INSTANTIATE_TEST_CASE_P(fusings_gpu, scatter_nd_update_scale_activation_eltwise,
+        gather_nd_test_params{ CASE_GATHER_ND_FP16_5D_6, 2, 3 },
+        gather_nd_test_params{ CASE_GATHER_ND_FP16_5D_7, 2, 3 },
+        gather_nd_test_params{ CASE_GATHER_ND_FP16_5D_8, 2, 3 },
+        gather_nd_test_params{ CASE_GATHER_ND_FP16_5D_9, 2, 3 },
+
+        gather_nd_test_params{ CASE_GATHER_ND_FP16_6D_1, 2, 3 },
+        gather_nd_test_params{ CASE_GATHER_ND_FP16_6D_2, 2, 3 },
+        gather_nd_test_params{ CASE_GATHER_ND_FP16_6D_3, 2, 3 },
+        gather_nd_test_params{ CASE_GATHER_ND_FP16_6D_4, 2, 3 },
+        gather_nd_test_params{ CASE_GATHER_ND_FP16_6D_5, 2, 3 },
+        gather_nd_test_params{ CASE_GATHER_ND_FP16_6D_6, 2, 3 },
+
+        gather_nd_test_params{ CASE_GATHER_ND_FP32_4D_1, 2, 3 },
+        gather_nd_test_params{ CASE_GATHER_ND_FP32_4D_2, 2, 3 },
+        gather_nd_test_params{ CASE_GATHER_ND_FP32_4D_3, 2, 3 },
+        gather_nd_test_params{ CASE_GATHER_ND_FP32_4D_4, 2, 3 },
+        gather_nd_test_params{ CASE_GATHER_ND_FP32_4D_5, 2, 3 },
+        gather_nd_test_params{ CASE_GATHER_ND_FP32_4D_6, 2, 3 },
+
+        gather_nd_test_params{ CASE_GATHER_ND_FP32_5D_1, 2, 3 },
+        gather_nd_test_params{ CASE_GATHER_ND_FP32_5D_2, 2, 3 },
+        gather_nd_test_params{ CASE_GATHER_ND_FP32_5D_3, 2, 3 },
+        gather_nd_test_params{ CASE_GATHER_ND_FP32_5D_4, 2, 3 },
+        gather_nd_test_params{ CASE_GATHER_ND_FP32_5D_5, 2, 3 },
+        gather_nd_test_params{ CASE_GATHER_ND_FP32_5D_6, 2, 3 },
+        gather_nd_test_params{ CASE_GATHER_ND_FP32_5D_7, 2, 3 },
+        gather_nd_test_params{ CASE_GATHER_ND_FP32_5D_8, 2, 3 },
+        gather_nd_test_params{ CASE_GATHER_ND_FP32_5D_9, 2, 3 },
+
+        gather_nd_test_params{ CASE_GATHER_ND_FP32_6D_1, 2, 3 },
+        gather_nd_test_params{ CASE_GATHER_ND_FP32_6D_2, 2, 3 },
+        gather_nd_test_params{ CASE_GATHER_ND_FP32_6D_3, 2, 3 },
+        gather_nd_test_params{ CASE_GATHER_ND_FP32_6D_4, 2, 3 },
+        gather_nd_test_params{ CASE_GATHER_ND_FP32_6D_5, 2, 3 },
+        gather_nd_test_params{ CASE_GATHER_ND_FP32_6D_6, 2, 3 },
+}), );
+
+class gather_nd_activation_scale_eltwise : public GatherNDPrimitiveFusingTest {};
+TEST_P(gather_nd_activation_scale_eltwise, basic) {
+    auto p = GetParam();
+
+    create_topologies(input_layout("input", get_input_layout(p)),
+        data("gather_nd_indices", get_mem(get_indices_layout(p), 0, p.max_number_in_indices - 1)),
+        data("scale_data", get_mem(get_per_channel_layout(p), -10, 10)),
+        data("eltwise_data", get_mem(get_output_layout(p))),
+        gather_nd("gather_nd_prim", "input", "gather_nd_indices", p.indices_rank, p.batch_dims),
+        activation("activation", "gather_nd_prim", activation_func::abs),
+        scale("scale", "activation", "scale_data"),
+        eltwise("eltwise", { "scale", "eltwise_data" }, eltwise_mode::sum, p.data_type),
+        reorder("reorder_bfyx", "eltwise", p.default_format, data_types::f32)
+    );
+
+    tolerance = 1.f;
```

Comment:
why 1.0?

Reply:
Just like https://github.com/openvinotoolkit/openvino/pull/4458,
I've tested with 1e-5 threshold but, some unittests are failed. I've tested with "-cl-opt-disable" and all unittests are passed with 1e-5. So I think this threshold issue comes from compiler-optimization. I will use tolerance=1.f until the compiler-optimization difference issue is solved.